edgelet/edgelet-core/Cargo.toml

```diff
@@ -14,16 +14,20 @@ futures = "0.1"
 failure = "0.1"
 humantime = "2"
 lazy_static = "1.0"
+nix = "0.23"
+num_cpus = "1.8.0"
 regex = "1"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
+serde_with = "1.0"
 sha2 = "0.7.0"
 log = "0.4"
 url = { version = "2", features = ["serde"] }
 url_serde = "0.2"
 tokio = "0.1"
 
+aziotctl-common = { git = "https://github.com/Azure/iot-identity-service", branch = "release/1.2" }
```

Comment:
why do we need this?

Reply:
`aziotctl-common` contains host-info-fetching routines like parsing `os-release`.